filesystem.go

```diff
@@ -173,11 +168,6 @@ func (fs *filesystem) storeFile(path string, data interface{}) error {
 }
 
 func (fs *filesystem) fetchFile(path string, data interface{}) error {
-	_, err := os.Stat(path)
```

Comment:
Yes, and `ioutil.Readfile()` will return an error if `path` doesn't exist, and crucially do it atomically rather than having to have 2 syscalls.